vips/image_golden_test.go

```diff
@@ -0,0 +1,221 @@
+package vips
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func TestImage_Resize_Downscale(t *testing.T) {
+	goldenTest(t, resources+"jpg-24bit.jpg", nil, func(img *ImageRef) error {
+		return img.Resize(0.9, KernelLanczos3)
+	})
+}
+
+func TestImage_Resize_Upscale(t *testing.T) {
+	goldenTest(t, resources+"jpg-24bit.jpg", nil, func(img *ImageRef) error {
+		return img.Resize(1.1, KernelLanczos3)
+	})
+}
+
+func TestImage_Resize_Downscale_Alpha(t *testing.T) {
+	goldenTest(t, resources+"png-8bit+alpha.png", nil, func(img *ImageRef) error {
+		return img.Resize(0.9, KernelLanczos3)
+	})
+}
+
+func TestImage_Resize_Upscale_Alpha(t *testing.T) {
+	goldenTest(t, resources+"png-8bit+alpha.png", nil, func(img *ImageRef) error {
+		return img.Resize(1.1, KernelLanczos3)
```

Comment:
maybe check the dimensions of the resultant image

Reply:
and for other tests as well

Reply:
Golden test, need to verify once the golden image